dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,66 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseOption):
```

Comment:
Should be a boolean option by base because it should have `is_enabled` as such should be initialized properly in init

Reply:
actually after further inspection should be based off of `BaseColumnOptions` which we can rename

Reply:
changed class name and inherits from that